_episodes/10-shielding.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Live Coding Shielding Problem"
+teaching: 0
+exercises: 90
+questions:
+- "How can I perform a shielding calculation using Python?"
+keypoints:
+- "Use Python to write a simple shielding calculation script."
+- "Learn some best practices when writing Python."
+objectives:
+- "Produce a Python script from scratch"
+- "Execute Python scripts in a command line"
+- "Selection of appropriate data structures for an application"
+- "Understand the value of using functions"
+- "Learn best practices for function implementation"
```

Comment:
"Learn" is not an outcome as it is actually just the action of getting to the outcome. So for this one, what we want them to come out with after the "learning" process is the learning outcome. So the actual learning outcome here is the implementation of the best practices:
"Implement functions as a software best practice."